marlowe-runtime-web/marlowe-runtime-web.cabal

```diff
@@ -54,31 +54,32 @@ library
   exposed-modules:
```

Comment:
Can you please run `cabal-fmt --inplace --no-tabular **/*.cabal && git commit -am "format cabal files" --no-verify`?